extra/3-magic-8-ball.js

```diff
@@ -58,10 +86,26 @@ function shakeBall() {
 
   This function should expect to be called with any value which was returned by the shakeBall function.
 */
-function checkAnswer(answer) {
-  //Write your code in here
+
+function checkAnswer(randomAnswer) {
+  for (i = 0; i < answers.length; i++) {
```

Comment:
What keyword do we need to put in front of a new variable?

Reply:
Question not answered.